### PR TITLE
Set maximum jobs to run on LSF

### PR DESCRIPTION
**Pull request by aef-**
This is just an idea, PR is not ready to be merged in.

### CODE REVIEW

File: toil_orchestrator/views/job_view.py [hunk 1]
```diff
@@ -24,7 +24,6 @@ def validate_and_save(self, data):
         serializer = JobSerializer(data=data)
         if serializer.is_valid():
             response = serializer.save()
-            submit_jobs_to_lsf.delay(str(response.id))
```
Comment (ionox0) [inline toil_orchestrator/views/job_view.py:27]: Just curious what this call would do previously. Does this mean when a job is updated in the admin it is re-submitted / re-run?

Comment (nikhil) [reply toil_orchestrator/views/job_view.py:27]: It would submit jobs after its submitted. I think this should be submit_pending_jobs instead.

**Review by nikhil — CHANGES_REQUESTED** [review]

File: toil_orchestrator/celery.py [hunk 2]
```diff
@@ -29,6 +28,11 @@
         "schedule": 60.0,
         "options": {"queue": settings.RIDGEBACK_DEFAULT_QUEUE}
     },
+    "check_status_of_jobs": {
```
Comment (aef-) [inline toil_orchestrator/celery.py:31]: change this name.

File: toil_orchestrator/views/job_view.py (hunk 1 above)
Comment (nikhil) [reply toil_orchestrator/views/job_view.py:27]: Nvm, I see this is handled by: https://github.com/mskcc/ridgeback/pull/151/files#diff-f828b61ce92a6fae16a7925b2fe809e9c72f91114cb5ac45df05b1530816bb7eR89.

**Review by nikhil — APPROVED** [review]

**Pull request merged**